### DISCUSSION TITLE
Elixir autoformatting sometimes is buggy

### DISCUSSION THREAD

**Discussion by emadb** [zed-industries/zed#22367] [Feature Requests]
Actually I don't know if this is a problem of zed or of the Elixir language server.
This is what happens when I write an anonymous function:

https://github.com/user-attachments/assets/af4f84ab-2ec0-4cf6-b23c-ecbdb4ada8ee

It's a little bit boring the autoformatting feature in this context.
Do you know if it a Zed bug or it's related to the Language Server (I'm using `elixir-ls`).

Thank you

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
